superset-frontend/src/pages/Login/index.tsx

```diff
@@ -0,0 +1,211 @@
+    });
+  };
+
+  return (
+    <LoginContainer justify="center" data-test="login-form">
+      <StyledCard title={t('Sign in')} padded>
+        {authType === AuthType.AuthOID && (
+          <Flex justify="center" vertical gap="middle">
+            <Form layout="vertical" requiredMark="optional" form={form}>
+              {providers.map((provider: OIDProvider) => (
+                <Form.Item<LoginForm>>
+                  <Button
+                    href={`/login/${provider.name}`}
+                    block
+                    iconPosition="start"
+                    icon={AuthIconMap[provider.name]}
+                  >
+                    {t('Sign in with')} {capitalize(provider.name)}
+                  </Button>
+                </Form.Item>
+              ))}
+            </Form>
+          </Flex>
+        )}
+        {authType === AuthType.AuthOauth && (
+          <Flex justify="center" gap={0} vertical>
+            <Form layout="vertical" requiredMark="optional" form={form}>
+              {providers.map((provider: OAuthProvider) => (
+                <Form.Item<LoginForm>>
+                  <Button
+                    href={`/login/${provider.name}`}
+                    block
+                    iconPosition="start"
+                    icon={AuthIconMap[provider.name]}
+                  >
+                    {t('Sign in with')} {capitalize(provider.name)}
+                  </Button>
+                </Form.Item>
+              ))}
+            </Form>
+          </Flex>
+        )}
+
+        {(authType === AuthType.AuthDB || authType === AuthType.AuthLDAP) && (
+          <Flex justify="center" vertical gap="middle">
+            <Typography.Text type="secondary">
+              {t('Enter your login and password below:')}
+            </Typography.Text>
+            <Form
+              layout="vertical"
+              requiredMark="optional"
+              form={form}
+              onFinish={onFinish}
+            >
+              <Form.Item<LoginForm>
+                label={<StyledLabel>{t('Username:')}</StyledLabel>}
+                name="username"
+                rules={[
+                  { required: true, message: t('Please enter your username') },
+                ]}
+              >
+                <Input
+                  prefix={<Icons.UserOutlined iconSize="l" />}
+                  data-test="username-input"
+                />
+              </Form.Item>
+              <Form.Item<LoginForm>
+                label={<StyledLabel>{t('Password:')}</StyledLabel>}
+                name="password"
+                rules={[
+                  { required: true, message: t('Please enter your password') },
+                ]}
+              >
+                <Input.Password
+                  prefix={<Icons.KeyOutlined iconSize="l" />}
+                  data-test="password-input"
+                />
+              </Form.Item>
+              <Form.Item label={null}>
+                <Flex
```

Comment:
You have a `LoginContainer` that is the same as this. I prefer inline, so maybe we can get rid of `LoginContainer`?